### PR TITLE
Introduce seed service

### PR DESCRIPTION
**Pull request by lukasaric**
This PR adds:

- Seed service
- `users` seed

### CODE REVIEW

File: server/services/seed.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Config } from '../config';
```
Comment (MiroDojkic) [inline server/services/seed.ts]: This logic is tied to the Database provider so should we move it to `shared/database/seed`?

File: server/shared/database/seeds/users.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import roles from '../../../user/roles';
+import times from 'lodash/times';
+import User from '../../../user/model';
+
+async function seedUsers(em: EntityManager): Promise<void> {
+  const users = times(10, idx => {
+    const suffix = idx || '';
+    const role = idx ? roles.USER : roles.ADMIN;
+    return new User(`User ${suffix}`, 'Example', `user${suffix}@example.org`, role);
+  });
```
Comment (MiroDojkic) [inline server/shared/database/seeds/users.ts]: This logic seems a bit hard to read. Why not separate admin and user seeding logic - we could have completely different credentials too, i.e. `admin@example.org`?

File: server/shared/database/seeds/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { EntityManager } from '@mikro-orm/postgresql';
```
Comment (MiroDojkic) [inline server/shared/database/seeds/index.ts]: Where is this used?

**Review by MiroDojkic — CHANGES_REQUESTED** [review]

File: server/shared/database/seeds/index.ts (hunk 3 above)
Comment (lukasaric) [reply server/shared/database/seeds/index.ts]: It's used below as `em` param  type.

File: server/shared/database/seeds/users.ts (hunk 2 above)
Comment (lukasaric) [reply server/shared/database/seeds/users.ts]: 👍

**Review by MiroDojkic — APPROVED** [review]

**Pull request merged**